lua/oil/config.lua

```diff
@@ -121,6 +121,10 @@ local default_config = {
     win_options = {
       winblend = 0,
     },
+    -- gap between oil window and preview window
+    preview_gap = 2,
+    -- preview_split: Split direction: "left", "right", "above", "below".
+    preview_split = 'left',
```

Comment:
I think the default should be `"auto"`, which choose left/right based on the value of `vim.o.splitright`